mobile/android/app/src/main/res/drawable-v21/launch_background.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What's the difference between `/drawable-v21/launch_background.xml` and `/drawable/launch_background.xml`?